src/mscorlib/src/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -142,7 +142,7 @@ private int GetHashCodeOfStringCore(string source, CompareOptions options)
             Debug.Assert(source != null);
             Debug.Assert((options & (CompareOptions.Ordinal | CompareOptions.OrdinalIgnoreCase)) == 0);
 
-            return GetHashCodeOfStringCore(source, options, forceRandomizedHashing: false, additionalEntropy: 0);
```

Comment:
Note on Unix, string hash codes were apparently not randomized. This changes them to randomized. Is that OK @jkotas ?

Reply:
> Note on Unix, string hash codes were apparently not randomized

Why do you think so? Do you have a repro?

Reply:
I see false on this line. I did not try it.